docs/api-docs/store-management/shipping/shipping-overview.md

```diff
@@ -1,52 +1,52 @@
 # Shipping
 <div class="otp" id="no-index">
-	<h3> On This Page </h3>
+	<h3> On this page </h3>
 	<ul>
     <li><a href="#shipping_storefront-server-checkout">Storefront Checkout and Server to Server Checkout</a></li>
-    <li><a href="#shipping_order-shipping-addresses">Order Shipping Addresses</a></li>
-    <li><a href="#shipping_shipping-zone-methods">Shipping Zone and Shipping Methods</a></li>
-    <li><a href="#shipping_real-time-carriers">Real-Time Carriers</a></li>
+    <li><a href="#shipping_order-shipping-addresses">Order shipping addresses</a></li>

```

Comment:
Since this title refers to an actual endpoint, I changed it back to title case. 
```suggestion
    <li><a href="#shipping_order-shipping-addresses">Order Shipping Addresses</a></li>

```